src/AspNet.Security.OAuth.Weibo/project.json

```diff
@@ -0,0 +1,45 @@
+﻿{
+  "version": "1.0.0-beta3-*",
+  "description": "ASP.NET Core middleware that enables an application to support Weibo's OAuth authentication workflow.",
+
+  "packOptions": {
+    "owners": [ "zhengchun" ],
+    "projectUrl": "https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers",
+    "iconUrl": "https://avatars3.githubusercontent.com/u/7998081?s=64",
+    "licenseUrl": "http://opensource.org/licenses/MIT",
```

Comment:
All the other providers use the Apache 2.0 license. Please change that to make sure we're consistent.